.circleci/configurations/jobs.yml

```diff
@@ -171,6 +171,8 @@ jobs:
                   (yarn test-e2e-ios 2>&1 | tee /tmp/test_log) || true
             - store_artifacts:
                 path: /tmp/test_log
+            - store_artifacts:
+                path: ~/react-native/packages/rn-tester-e2e/reports
```

Comment:
You can also include the error screenshots here. That way, the engineer can see a screenshot of the failure.